ngraph/frontend/paddlepaddle/include/paddlepaddle_frontend/place.hpp

```diff
@@ -1,5 +1,5 @@
 //*****************************************************************************
-// Copyright 2017-2021 Intel Corporation
```

Comment:
Better to change file header to new format

Reply:
I don't really know which format is better. We use this header everywhere in the frontend directory.

Reply:
Seems it was used in 2020 (when PoC was actually created), but recently it has been changed everywhere in other code on master to short header. So, we should do the same, I guess sooner or later

Reply:
will be done in the next PR